Transient IT failure in GitLab `anvildev` for `develop` pipeline run

https://gitlab.anvil.gi.ucsc.edu/ucsc/azul/-/jobs/9838#L1232
```
AIL: test_indexing (integration_test.IndexingIntegrationTest) [other_endpoints] (endpoint=furl('https://indexer.anvil.gi.ucsc.edu'/), path='/health', args=None)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builds/ucsc/azul/test/integration_test.py", line 378, in subTest
    yield
  File "/builds/ucsc/azul/test/integration_test.py", line 535, in _test_other_endpoints
    self._check_endpoint(path, args=args, endpoint=endpoint)
  File "/builds/ucsc/azul/test/integration_test.py", line 630, in _check_endpoint
    return self._get_url_content(url)
  File "/builds/ucsc/azul/test/integration_test.py", line 636, in _get_url_content
    return self._get_url(url).data
  File "/builds/ucsc/azul/test/integration_test.py", line 648, in _get_url
    self._assertResponseStatus(response, expected_statuses)
  File "/builds/ucsc/azul/test/integration_test.py", line 673, in _assertResponseStatus
    assert response.status in expected_statuses, (
AssertionError: (504, 'Gateway Timeout', b'{"message": "Endpoint request timed out"}')
----------------------------------------------------------------------
```

--- a/src/azul/bigquery_reservation.py
+++ b/src/azul/bigquery_reservation.py
@@ -5,9 +5,6 @@ from typing import (
     Union,
 )
 
-from google.auth.transport.urllib3 import (
-    AuthorizedHttp,
-)
 from google.cloud.bigquery_reservation_v1 import (
     Assignment,
     CapacityCommitment,
@@ -25,7 +22,6 @@ from google.oauth2.service_account import (
 from more_itertools import (
     one,
 )
-import urllib3
 
 from azul import (
     cached_property,
@@ -34,9 +30,6 @@ from azul import (
 )
 from azul.deployment import (
     aws,
-)
-from azul.http import (
-    http_client,
 )
 
 log = logging.getLogger(__name__)
@@ -93,10 +86,6 @@ class BigQueryReservation:
     @cached_property
     def _client(self) -> ReservationServiceClient:
         return ReservationServiceClient(credentials=self.credentials)
-
-    @cached_property
-    def _http_client(self) -> urllib3.PoolManager:
-        return AuthorizedHttp(self.credentials, http_client())
 
     @property
     def _project(self) -> str:

--- a/src/azul/health.py
+++ b/src/azul/health.py
@@ -10,6 +10,7 @@ from itertools import (
     chain,
 )
 import json
+import logging
 import time
 from typing import (
     ClassVar,
@@ -58,6 +59,8 @@ from azul.types import (
     JSON,
 )
 
+log = logging.getLogger(__name__)
+
 
 # noinspection PyPep8Naming
 class health_property(cached_property):
@@ -66,6 +69,10 @@ class health_property(cached_property):
     returned by HealthController.as_json(). Be sure to provide a docstring in
     the decorated method.
     """
+
+    def __get__(self, obj, objtype=None):
+        log.info('Getting health property %r', self.key)
+        return super().__get__(obj, objtype=objtype)
 
     @property
     def key(self):
@@ -233,6 +240,7 @@ class Health:
 
     def _api_endpoint(self, relative_url: furl) -> tuple[str, JSON]:
         url = str(config.service_endpoint.join(relative_url))
+        log.info('Requesting %r', url)
         response = requests.head(url)
         try:
             response.raise_for_status()
@@ -273,6 +281,7 @@ class Health:
         try:
             url = config.lambda_endpoint(lambda_name).set(path='/health/basic',
                                                           args={'catalog': self.catalog})
+            log.info('Requesting %r', url)
             response = requests.get(str(url))
             response.raise_for_status()
             up = response.json()['up']

